application/org.openjdk.jmc.flightrecorder.flameview/src/main/resources/page.template

```diff
@@ -10,15 +10,28 @@
 	<![endif]-->
 </head>
 
-<body onresize="resizeFlameGraph()">
+<body>
+	<div id="search" align="right">
+		<input type="text" id="packageNameInput" size="22" onfocusout="executeSearch()" value="">
+		<img src="data:image/png;base64, %3$s" width="12" height="12">
+	</div>
 	<div id="chart"></div>
-	<script type="text/javascript">%3$s</script>
 	<script type="text/javascript">%4$s</script>
+	<script type="text/javascript">%5$s</script>
 	<script type="text/javascript">
-	
+		
+		var textToSearch = "";
 		var flameGraph;
 		var currentJson;
 		
+		const packageInputField = document.getElementById("packageNameInput");
+		packageInputField.addEventListener("keyup", function(event) {
+				if(event.keyCode === 13){
```

Comment:
formatting: space after `if` and before `{`

Reply:
update